net/ddns-scripts/Makefile

```diff
@@ -61,6 +62,7 @@ define Package/ddns-scripts-cloudflare
   $(call Package/ddns-scripts/Default)
   TITLE:=Extension for cloudflare.com API v4
   DEPENDS:=ddns-scripts +curl
+  PROVIDES:=ddns-scripts_digitalocean.com-v2
```

Comment:
This should be: ``ddns-scripts_cloudflare.com-v4``

Reply:
oops, I am not sure how that happened, fixed in #19628